packages/opentelemetry-instrumentation/src/types.ts

```diff
@@ -44,6 +44,12 @@ export interface Instrumentation {
   /** Method to set meter provider  */
   setMeterProvider(meterProvider: MeterProvider): void;
 
+  /** Method to set instrumentation config  */
+  setConfig(config: InstrumentationConfig & any): void;
```

Comment:
all instrumentation's config should extends `InstrumentationConfig` so i don't think this should extends any,